### ISSUE TITLE
Support CloudWatch percentiles

### ISSUE THREAD

**Issue by hjacobs** [zalando-zmon/zmon-worker#190]
https://aws.amazon.com/about-aws/whats-new/2016/11/amazon-cloudwatch-adds-support-for-percentiles/

**Comment by Jan-M** [comment]
This may help for the unfamiliar ones: 
https://github.com/zalando-zmon/zmon-worker/blob/master/zmon_worker_monitor/builtins/plugins/cloudwatch.py

**Comment by harti2006** [comment]
I just played around and noticed, that the API for percentiles is a bit different. You have to pass the percentiles, that you are interested in, as "ExtendedStatistics" parameter.
In the response, the percentile values are also wrapped in a field named like this. See http://boto3.readthedocs.io/en/latest/reference/services/cloudwatch.html#CloudWatch.Client.get_metric_statistics for details.

**Comment by gamsd** [comment]
`Support for percentiles on metrics from Elastic Load Balancers or from your existing Application Load Balancers will be available automatically within the next few weeks.` It seems that the next few weeks still haven't arrived for my classic load balancers. Probably this won't be immediately useful for everyone, as it won't be for me in the end, but I hope it helps the users of ApplicationLBs at least.

Note that I changed the `query_one` method only. I didn't change the `query` method. I tried a quite simple implementation, but wasn't sure about keeping it there, so in benefit of smaller changes I removed it from the PR. Also, as a user, I'm more concerned with `query_one`. If you think it's worth to also update the `query` method already, we can discuss it further and I can do it after some initial feedback on #193.
